documentation/docs/developers/building-and-deploying-to-ios.md

```diff
@@ -0,0 +1,59 @@
+# Building and deploying to iOS
+
+Building and deploying an app to iOS must be performed by a developer on a Mac computer. This guide does not include the process for managing the app within Apple's App Store Connect console, which is documented elsewhere.
+
```

Comment:
nit(non-blocking): not sure what you mean by 'elsewhere' - in these documents or just generally 'not here'?
If implying 'not here' then probably better to direct to another info source, e.g. the apple store help docs:
https://developer.apple.com/help/app-store-connect/